config/configgrpc/configgrpc_test.go

```diff
@@ -536,3 +536,27 @@ func tempSocketName(t *testing.T) string {
 	require.NoError(t, os.Remove(socket))
 	return socket
 }
+
+func TestGRPCPortReuse(t *testing.T) {
+	settings := GRPCServerSettings{
+		NetAddr: confignet.NetAddr{
+			Endpoint:  "localhost: 2873",
+			Transport: "tcp",
+		},
+	}
+	muxListener, err := settings.ToListener()
+	if err != nil {
+		t.Errorf("cannot initialize multiplexing listener")
+	}
+
+	for i := 0; i < 100; i++ {
```

Comment:
Does this have to be 100 or can it be something smaller?

Reply:
I think anything reasonable is ok.